docs/dev/api/rdc.md

```diff
@@ -1322,3 +1331,387 @@ No payload is returned with the successful deletion.
 </details>
 
 ---
+
+## Private Real Device Management
+
+Learn more about how to [manage your private devices](/basics/acct-team-mgmt/private-device-mgmt).
+
+### Get Private Devices
+<details>
+<summary><span className="api get">GET</span> <code>/v1/rdc/device-management/devices</code></summary>
+<p/>
+
+Get a list of private devices with their device information and settings.
+
+#### Parameters
+
+This method takes no parameters.
+
+<Tabs
+groupId="dc-url"
+defaultValue="us"
+values={[
+{label: 'United States', value: 'us'},
+{label: 'Europe', value: 'eu'},
+]}>
+
+<TabItem value="us">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.us-west-1.saucelabs.com/v1/rdc/device-management/devices' | json_pp
+```
+
+</TabItem>
+
+<TabItem value="eu">
+
+```jsx title="Sample Request"
+curl -u "$SAUCE_USERNAME:$SAUCE_ACCESS_KEY" --location \
+--request GET 'https://api.eu-central-1.saucelabs.com/v1/rdc/device-management/devices' | json_pp
+```
+
+</TabItem>
+</Tabs>
+
+#### Responses
+
+<table id="table-api">
+<tbody>
+  <tr>
+    <td><code>200</code></td>
+    <td colSpan='2'>Success. List of private devices is returned.</td>
+  </tr>
+</tbody>
+</table>
+
+```jsx title="Sample Response"
+[
+    {
+        "id": "iPhone_12_17_sl",
+        "name": "iPhone 12",
+        "os": {
+            "name": "iOS",
+            "version": "17.6"
+        },
+        "screenSize": 6.1,
+        "resolutionWidth": 1170,
+        "resolutionHeight": 2532,
+        "state": "AVAILABLE",
+        "team": null,
+        "appWhitelist": [
+            "com.google.chrome.ios"
+        ],
+        "accountWhitelist": [],
+        "systemAppAllowlist": [],
+        "applePaySupportEnabled": false,
+        "skipCleaningFolders": []
+    },
+    {...more devices},
+]
+```
+
+</details>
+
+---
+
+### Assign Device to a Team
+<details>
+<summary><span className="api put">PUT</span> <code>/v1/rdc/device-management/devices/&#123;device_id&#125;/team</code></summary>
+<p/>
+
+Assign a private device to a specific team.
+
+#### Parameters
+
+<table id="table-api">
+  <tbody>
+    <tr>
+     <td><code>device_id</code></td>
```

Comment:
@yuyi-sl  can you add an example how this ID looks like? 
iphone_15_blabla?

Reply:
Added (Example: iphone_12_17_sl) to all device_id parameter info for device management

Reply:
thank you